src/baldr/graphtile.cc

```diff
@@ -571,33 +571,38 @@ AABB2<PointLL> GraphTile::BoundingBox() const {
 }
 
 iterable_t<const DirectedEdge> GraphTile::GetDirectedEdges(const NodeInfo* node) const {
```

Comment:
changed all these to:

1. logic_error because its definitly that vs a runtime_error
2. put the error check at the top of the method in the if block so the default is not branch inward